### ISSUE TITLE
Why require so many timestamps?

### ISSUE THREAD

**Issue by xeruf** [novoid/lazyblorg#103]
Why does a lazyblorg post require a CREATED and a MODIFIED property, if we can already deduce these from the logbook?
This feels annoyingly redundant.
Would be good to document these requirements as well in the steps of the readme, and which timestamp is used for which purpose.

**Comment by novoid** [comment]
Good point. Somewhere I think I summarized the reasoning but that can be made more accessible.

In short: CREATED is created automatically in my setup so it's no effort on my side.

I don't think that MODIFIED is part of lazyblorg.

**Comment by xeruf** [comment]
true, it requires CREATED and a logbook - my setup unfortunately depends on the used file at the moment

**Comment by novoid** [comment]
What do you mean "on the used file"?

**Comment by xeruf** [comment]
I have different org defaults for different org files.

**Comment by novoid** [comment]
The parser parses it and stores it under `entry_data['created']`.

`lib/htmlizer.py` doesn't seem to use it for anything at the moment.

It looks to me that the parser is expecting it (and reports an error if missing) but it is not used. As a test, you could remove the corresponding check in `__check_if_entry_is_OK()` (4 lines) and do a test-run if everything is working. If so, report back and I'll think about removing it upstream as well.
